<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="8" passed="6" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-08T09:35:35Z" name="Default suite" finished-at="2021-08-08T09:37:23Z" duration-ms="107330">
    <groups>
    </groups>
    <test started-at="2021-08-08T09:35:35Z" name="Default test" finished-at="2021-08-08T09:37:23Z" duration-ms="107330">
      <class name="ValidatingLandingPage.ValidatingMoreTravel">
        <test-method is-config="true" signature="setUp()[pri:0, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:35:35Z" name="setUp" finished-at="2021-08-08T09:36:43Z" duration-ms="67947" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testSize()[pri:11, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:36:43Z" name="testSize" finished-at="2021-08-08T09:36:59Z" duration-ms="15996" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSize -->
        <test-method signature="testValues(java.lang.String)[pri:12, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:37:08Z" name="testValues" data-provider="data" finished-at="2021-08-08T09:37:09Z" duration-ms="793" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Stays]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValues -->
        <test-method signature="testValues(java.lang.String)[pri:12, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:37:09Z" name="testValues" data-provider="data" finished-at="2021-08-08T09:37:09Z" duration-ms="668" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Flights]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValues -->
        <test-method signature="testValues(java.lang.String)[pri:12, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:37:09Z" name="testValues" data-provider="data" finished-at="2021-08-08T09:37:11Z" duration-ms="1228" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Cars]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValues -->
        <test-method signature="testValues(java.lang.String)[pri:12, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:37:11Z" name="testValues" data-provider="data" finished-at="2021-08-08T09:37:11Z" duration-ms="8" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Holiday Packages]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at ValidatingLandingPage.ValidatingMoreTravel.testValues(ValidatingMoreTravel.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValues -->
        <test-method signature="testValues(java.lang.String)[pri:12, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:37:11Z" name="testValues" data-provider="data" finished-at="2021-08-08T09:37:11Z" duration-ms="2" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Holiday activity]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at ValidatingLandingPage.ValidatingMoreTravel.testValues(ValidatingMoreTravel.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValues -->
        <test-method signature="testValues(java.lang.String)[pri:12, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:37:11Z" name="testValues" data-provider="data" finished-at="2021-08-08T09:37:11Z" duration-ms="785" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Deals]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValues -->
        <test-method signature="testValues(java.lang.String)[pri:12, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:37:11Z" name="testValues" data-provider="data" finished-at="2021-08-08T09:37:12Z" duration-ms="797" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Mobile]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValues -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:ValidatingLandingPage.ValidatingMoreTravel@5e3d57c7]" started-at="2021-08-08T09:37:12Z" name="tearDown" finished-at="2021-08-08T09:37:23Z" duration-ms="10284" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- ValidatingLandingPage.ValidatingMoreTravel -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
